lib/fog/vsphere/requests/compute/list_vm_interfaces.rb

```diff
@@ -62,10 +62,17 @@ def get_raw_interface(vm_id, options = {})
         private
 
         def raw_to_hash(nic)
+          if nic.backing.respond_to?('network')
```

Comment:
While we're at it, can you please change `'network'` to a symbol (`:network`)? Same for port below.